src/compas_timber/connections/utilities.py

```diff
@@ -164,20 +162,20 @@ def are_beams_coplanar(beam_a, beam_b, tol=TOL):
     bool
         True if the beams are coplanar, False otherwise.
     """
+    if len(beams) != 2:
```

Comment:
if this function only supports 2 beams, why not keep it explicit by keeping the previous signature?